src/main/java/org/jenkinsci/plugins/ownership/integrations/securityinspector/PermissionsForOwnerReportBuilder.java

```diff
@@ -0,0 +1,219 @@
+ * of this software and associated documentation files (the "Software"), to deal
+ * in the Software without restriction, including without limitation the rights
+ * to use, copy, modify, merge, publish, distribute, sublicense, and/or sell
+ * copies of the Software, and to permit persons to whom the Software is
+ * furnished to do so, subject to the following conditions:
+ *
+ * The above copyright notice and this permission notice shall be included in
+ * all copies or substantial portions of the Software.
+ *
+ * THE SOFTWARE IS PROVIDED "AS IS", WITHOUT WARRANTY OF ANY KIND, EXPRESS OR
+ * IMPLIED, INCLUDING BUT NOT LIMITED TO THE WARRANTIES OF MERCHANTABILITY,
+ * FITNESS FOR A PARTICULAR PURPOSE AND NONINFRINGEMENT. IN NO EVENT SHALL THE
+ * AUTHORS OR COPYRIGHT HOLDERS BE LIABLE FOR ANY CLAIM, DAMAGES OR OTHER
+ * LIABILITY, WHETHER IN AN ACTION OF CONTRACT, TORT OR OTHERWISE, ARISING FROM,
+ * OUT OF OR IN CONNECTION WITH THE SOFTWARE OR THE USE OR OTHER DEALINGS IN
+ * THE SOFTWARE.
+ */
+package org.jenkinsci.plugins.ownership.integrations.securityinspector;
+
+import hudson.Extension;
+import hudson.model.Computer;
+import hudson.model.Descriptor;
+import hudson.model.Hudson;
+import hudson.model.Item;
+import hudson.model.TopLevelItem;
+import hudson.model.User;
+import hudson.model.View;
+import hudson.security.Permission;
+import hudson.security.PermissionGroup;
+import java.util.HashSet;
+import java.util.List;
+import java.util.Set;
+import java.util.regex.Pattern;
+import java.util.regex.PatternSyntaxException;
+import javax.annotation.Nonnull;
+import javax.servlet.ServletException;
+import org.acegisecurity.Authentication;
+import org.acegisecurity.context.SecurityContext;
+import org.acegisecurity.context.SecurityContextHolder;
+import org.acegisecurity.userdetails.UsernameNotFoundException;
+import org.jenkinsci.plugins.securityinspector.Messages;
+import static org.jenkinsci.plugins.securityinspector.SecurityInspectorAction.getSessionId;
+import org.jenkinsci.plugins.securityinspector.UserContext;
+import org.jenkinsci.plugins.securityinspector.UserContextCache;
+import org.jenkinsci.plugins.securityinspector.impl.users.UserReportBuilder;
+import org.jenkinsci.plugins.securityinspector.model.PermissionReport;
+import org.jenkinsci.plugins.securityinspector.model.SecurityInspectorReport;
+import org.jenkinsci.plugins.securityinspector.util.JenkinsHelper;
+import org.kohsuke.accmod.Restricted;
+import org.kohsuke.accmod.restrictions.NoExternalUse;
+import org.kohsuke.stapler.HttpResponses;
+import org.kohsuke.stapler.StaplerRequest;
+
+@Extension(optional = true)
+public class PermissionsForOwnerReportBuilder extends UserReportBuilder {
+    @Override
+    public String getIcon() {
+        return "fingerprint.png";
+    }
+
+    @Override
+    public String getIndex() {
+        return "items-for-owner";
+    }
+
+    @Override
+    public String getDisplayName() {
+        return "Single owner, multiple jobs";
```

Comment:
It also seems that the report also works for folders and other Items, not only for jobs